lib/room.rb

```diff
@@ -1,12 +1,22 @@
-class Rom
+class Room
 
+  attr_reader :category
   def initialize(category, length, width)
     @category = category
-    @length = 10
-    @width = width
+    @length = length
+    @width = width.to_i
+    @painted = false
   end
 
   def area
     @length * @width
   end
+
+  def is_painted?
+    @painted
+  end
+
+  def paint
+    @painted = true
+  end
```

Comment:
very nice work with this class! Good job working with the boolean method and catching that you needed to convert the width to an integer.